roles/sap_vm_temp_vip/README.md

```diff
@@ -1,81 +1,78 @@
-`Beta`
-
+<!-- BEGIN Title -->
 # sap_vm_temp_vip Ansible Role
+<!-- END Title -->
 
-Ansible Role for assignment of Temporary Virtual IP (VIP) to OS Network Interface prior to Linux Pacemaker ownership.
-
-This Ansible Role will (dependent on detected Infrastructure Platform) perform assignment of a Virtual IP Address to the OS Network Interface.
-
-
-## Functionality
-
-The hosts for SAP Software allocated for High Availability are configured with a temporary Virtual IP for the OS Network Interface; thereby allowing Linux Pacemaker to be installed once the SAP Software installation has concluded (best practice for Linux Pacemaker). When an Infrastructure Platform with specific requirements is detected (e.g. Load Balancers), then bespoke actions are performed.
-
-
-## Scope
+## Description
+<!-- BEGIN Description -->
+Ansible role `sap_vm_temp_vip` is used to enable installation of SAP Application and Database on High Availability clusters provisioned by [sap_vm_provision](https://github.com/sap-linuxlab/community.sap_infrastructure/tree/main/roles/sap_vm_provision) role.
 
-Only hosts required for High Availability (such as SAP HANA Primary node, SAP NetWeaver ASCS/ERS) should use this Ansible Role.
+Installation of cluster environment requires temporary assignment of Virtual IP (VIP) before executing installation roles [sap_hana_install](https://github.com/sap-linuxlab/community.sap_install/tree/main/roles/sap_hana_install) and [sap_swpm](https://github.com/sap-linuxlab/community.sap_install/tree/main/roles/sap_swpm).
+- This is temporary and it will be replaced by Cluster VIP resource once cluster is configured by [sap_ha_pacemaker_cluster](https://github.com/sap-linuxlab/community.sap_install/tree/main/roles/sap_ha_pacemaker_cluster) role.
 
-Assumptions are made based upon the default High Availability configuration for a given Infrastructure Platform (e.g. using Linux Pacemaker `IPAddr2` resource agent).
+This role does not update `/etc/hosts` or DNS, because that is completed by [sap_vm_provision](https://github.com/sap-linuxlab/community.sap_infrastructure/tree/main/roles/sap_vm_provision) role.
+<!-- END Description -->
 
+## Prerequisites
+<!-- BEGIN Prerequisites -->
+Environment:
+1. Execute role with `gather_facts: true`, as it uses `ansible_default_ipv4` facts.
+2. Assign hosts to correct groups, which are also used in other roles in our project
+    - Supported cluster groups: `hana_primary, hana_secondary, anydb_primary, anydb_secondary, nwas_ascs, nwas_ers`
 
-## Requirements
-
-### Target hosts
-
-**OS Versions:**
-- Red Hat Enterprise Linux 8.2+
-- SUSE Linux Enterprise Server 15 SP3+
-
-### Execution/Controller host
-
-**Dependencies:**
-- OS Packages
-  - Python 3.9.7+ (i.e. CPython distribution)
-- Python Packages
-    - None
-- Ansible
-    - Ansible Core 2.12.0+
-    - Ansible Collections:
-      - None
-
+Dependent on roles:
```

Comment:
Do you mean `Dependent roles`?

In this case, I suggest to modify the following line to `, for creating` instead of `to create`, as in:
` [sap_vm_provision](https://github.com/sap-linuxlab/community.sap_infrastructure/tree/main/roles/sap_vm_provision), for creating the following required resources: DNS, Load Balancers and Health Checks.`

Reply:
Thank you, changes were done in latest commit.